tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -516,3 +534,34 @@ Check And Uninstall Operator In Openshift
         END
     END
     Close All Browsers
+
+Validate URls
+    [Documentation]   validates  urls
+    [Arguments]  ${urls}
+    FOR  ${url}   IN   ${urls}
+        Log To Console  ${url}
+        ${status}=    Check HTTP Status Code    link_to_check=${url}
+        Log To Console     ${url}  gets status code ${status}
+    END
+
+Get Urls and validation of quickStrat Tiles
+    [Documentation]     Clicks on the side window and  validate all Url
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
+
+            ${status}   Run Keyword And Return Status   Click Element  ${sideWindowButton}
+            IF  ${status} == False
+                Click Button    Next
+            END
+
+            FOR    ${counter}    IN RANGE    5
+                Press Keys    NONE    TAB
+            END
+            Wait Until Element Is Visible  //button[@class='pf-c-wizard__nav-link']
+
+            # get all urls from the  side windows
+            ${status}   ${urls}    Run Keyword And Ignore Error   Get Element Attribute    //div[@Class="pf-c-drawer__panel-main"]//a[@rel="noopener noreferrer"]    href
```

Comment:
I think it is missing some links. For example, in some Seldon's quick start, in certain steps there should be 2 links, while the code captures only the first one. Maybe you should do something like `Get Web Elements` before getting the elemet attribute